src/utils/helpers/formatTime.ts

```diff
@@ -0,0 +1,14 @@
+import moment from "moment";
+
+export const CARD_TIME_FORMAT = "DD.MM.YYYY/ h:mm A";
```

Comment:
all timestamps in the database are iso-8601, this format is not supported anywhere